web/ui/dashboard/src/app/private/pages/project/project.component.html

```diff
@@ -1,61 +1,53 @@
-<div class="page layout__sidebar flex padding-left__24px padding-right__24px padding-bottom__40px">
-	<aside class="padding-top__40px border__right position__relative" [ngClass]="{ full: shouldShowFullSideBar }">
-		<button
-			[disabled]="screenWidth < 1150"
-			class="sidebar--button button__clear rounded__50px border__all width__32px height__32px flex flex__align-items-center flex__justify-center position__absolute position--top__8px bg__white"
-			(click)="shouldShowFullSideBar = !shouldShowFullSideBar"
+<convoy-page className="flex pt-0 pb-0">
+	<aside class="w-fit pt-10 border-r border-r-grey-10 relative">
+		<convoy-button
+			type="icon"
+			buttonType="button"
+			className="px-0 py-0 border border-grey-10 bg-white-100 rounded-100px w-[32px] h-[32px] absolute top-[8px] -right-[16px]"
+			[disable]="screenWidth < 1150"
+			(clickItem)="shouldShowFullSideBar = !shouldShowFullSideBar"
 		>
-			<svg width="12" height="12" fill="var(--primary-color)" class="open-icon">
+			<svg width="12" height="12" class="fill-primary-100 transition-all duration-200" [class]="shouldShowFullSideBar ? 'hidden' : 'block'">
 				<use xlink:href="#sidebar-icon"></use>
 			</svg>
-			<svg width="12" height="12" fill="var(--primary-color)" class="close-icon">
+			<svg width="12" height="12" class="fill-primary-100 transition-all duration-200" [class]="!shouldShowFullSideBar ? 'hidden' : 'block'">
 				<use xlink:href="#sidebar-icon-2"></use>
 			</svg>
-		</button>
-
-		<nav class="sidebar--content padding-left__8px padding-right__24px">
+		</convoy-button>
+		<nav class="pl-[8px] pr-[24px]">
 			<ul>
-				<li *ngFor="let nav of sideBarItems" class="animate">
-					<a [routerLink]="'.' + nav.route" routerLinkActive="active" class="flex flex__align-items-center color__grey padding-y__8px padding-left__24px margin-y__8px rounded__8px animate">
-						<svg
-							width="18"
-							height="18"
-							class="margin-right__10px"
-							[attr.xlink:fill]="nav.icon == 'subscriptions' ? '' : 'var(--grey-color)'"
-							[attr.stroke]="nav.icon == 'subscriptions' ? 'var(--grey-color)' : ''"
-							[id]="nav.icon"
-						>
+				<li [class]="shouldShowFullSideBar ? 'w-[182px]' : 'w-[66px]'" *ngFor="let nav of sideBarItems" class="animate">
```

Comment:
You don't need arbitrary values here, just `w-182px` and `w-66px` is fine, see `pl-[8px] pr-[24px]` above too.

See the tailwind config, space values are in multiples of two.